app/src/main/java/com/jHerscu/clearskies/data/source/local/CityDB.kt

```diff
@@ -0,0 +1,9 @@
+package com.jHerscu.clearskies.data.source.local
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+
+@Database(entities = [GeocodedCity::class], version = 1, exportSchema = false)
+abstract class CityDB : RoomDatabase() {
```

Comment:
there is no hard and fast rule or anything, but generally you'd want just one DB for your application with multiple DAOs in it

Reply:
Thanks, I'm glad I got it in front of you early before I went any further. I just learned how to link multiple tables in a db last night, but I was unsure how many entities should be included or if unrelated entities could exist in the same db.

Reply:
From your answer, I'm guessing that any needed tables would go into one database, regardless of amount or relation.

Reply:
@j-roskopf would I consolidate the dao's as well?

Reply:
not necessarily! again, there is no hard and fast rule or anything, but generally what you see in the wild is one database class that has multiple DAOs grouped by feature / similarity. in regards to your weather app, it would make sense to me to have a 'ClearSkies' db and a weather and city DAO living in that DB.